source/DataAccess/Oracle/OracleDDLHelper.ts

```diff
@@ -16,12 +16,17 @@ export default class OracleDDLHelper implements IDDLHelper {
                 return validTableNames.indexOf(name) > -1;
             })
             .map((name) => {
-                return `SELECT 1 as "priority", \'${name}\' as table_name, DBMS_METADATA.GET_DDL(\'TABLE\', \'${name}\') as ddl ` +
-                'FROM dual ' +
-                'union all ' +
-                // We know what tables exist but must discover any indexes
-                `SELECT 2 as "priority", \'${name}\' as table_name, DBMS_METADATA.GET_DDL(\'INDEX\', uidx.INDEX_NAME) as ddl ` +
-                `FROM USER_INDEXES uidx where table_name = '${name}' `;
+                // This apparently does not return metadata for some system tables/views like ALL_TABLES but will return
```

Comment:
This is becoming increasingly frustrating because of Oracles various potential ways to query for valid, user accessible tables. I am no longer confident in producing the indexes at this time so that has been commented out for future posterity. The current known strategy that returns something other than an empty result set (looking at you USER_TABLES!) is to hit ALL_TABLES. There is no guarantee this will work elsewhere but for now it's the only known working option.